ch03tests/05Mocks.ipynb

```diff
@@ -359,14 +210,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 3. Examples "
+    "## Examples "
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### 3.1 Using mocks to model test resources\n",
+    "### Using mocks to model test resources\n",
     "Often we want to write tests for code which interacts with remote resources. (E.g. databases, the internet, or data files.)\n",
     "We don't want to have our tests *actually* interact with the remote resource, as this would mean our tests failed\n",
```

Comment:
```suggestion
    "We don't want to have our tests *actually* interact with the remote resource, as this would mean our tests could fail unpredictably from time to time, through no fault of our own, \n",
```
mini-improvement, while we're here :)